code/script.js

```diff
@@ -1,47 +1,184 @@
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Welcome stranger! What's your name?`, "bot")
+}
+const userReply = (input) => {
+  showMessage(input, "user")
+}
+sendBtn.addEventListener("click", () => {
+  userReply(userInput.value)
+  setTimeout(pickDestination, 1000)
+})
+
+// Question 2
+
+const pickDestination = () => {
+  showMessage(`Nice to meet you ${userInput.value}! Choose one of the below options!`, "bot")
+  inputWrapper.innerHTML = `
+  <button id= "beachBtn"> Beach </button>
+  <button id= "skiBtn"> Ski </button>
+  <button id= "cityBtn"> City </button>
+  `
+  const beachBtn = document.getElementById("beachBtn")
+  const skiBtn = document.getElementById("skiBtn")
+  const cityBtn = document.getElementById("cityBtn")
+  beachBtn.onclick = () => {
+
+    //when user click on the Ski button, it's going to change background img to snowimage.
+    
+    document.body.style.backgroundImage = "url('IMG/beach.jpg')";
+    userReply(`beach`)                 //invoke userReply function with the input "beach"
+    setTimeout(pickTravelDuration, 1000)  // After 1s invoke function pickTravelDuration
+  }
+  skiBtn.onclick = () => {
+    document.body.style.backgroundImage = "url('IMG/sky.jpg')";
+    userReply(`ski`)
+    setTimeout(pickTravelDuration, 1000)
+  }
+  cityBtn.onclick = () => {
+    document.body.style.backgroundImage = "url('IMG/city.jpg')";
+    userReply(`city`)
+    setTimeout(pickTravelDuration, 1000)
+  }
+} 
+
+// Question 3
+
+const pickTravelDuration = () => {
+  showMessage(`Nice choise! Now choose how far you want to travel?`, "bot")
+  inputWrapper.innerHTML = `
+  <button id= "lessBtn"> Less than 8 hours </button>
+  <button id= "moreBtn"> More than 8 hours </button>
+  `
+  const lessBtn = document.getElementById("lessBtn")
+  const moreBtn = document.getElementById("moreBtn")
+  lessBtn.onclick = () => {
+    inputWrapper.innerHTML = `
+      <select id="selectLess" onchange="updateLess()" >
+        <option value="" selected disabled>When do you want to leave?</option>
+        <option id= "week2" value="week2">In 1 week</option>
+        <option id= "month2" value="month2">In 1 month</option>
+        <option id= "months2" value="months2">In 3 months</option>
+      </select>
+      `
+    userReply(`Less than 8 hours`)
+  }
+  moreBtn.onclick = () => {
+    inputWrapper.innerHTML = `
+      <select id="selectMore" onchange= "updateMore ()">
+        <option value="" selected disabled>When do you want to leave?</option>
+        <option id= "week" value="week">In 1 week</option>
+        <option id= "month" value="month">In 1 month</option>
+        <option id= "months" value="months">In 3 months</option>
+      </select>
+      `
+    userReply(`More than 8 hours`)
+  }
+}
+
+// Question 4
+
+const showTravelTime = () => {
+  showMessage(`Great!`, "bot")
+  setTimeout(() => showMessage(`Unfortunately it´s fully booked but we will send you here instead! \uD83C\uDF32 \uD83C\uDF32
```

Comment:
An idea here would maybe be to make the chat stop after the last question with either refresh the page or just break. Now you can make an option several times.